sections/footer.liquid

```diff
@@ -247,7 +248,7 @@
   {%- endif -%}
 
   <div
-    class="footer__content-bottom{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}"
+    class="footer__content-bottom"
```

Comment:
@Roi-Arthur thanks for the suggestion! I explored your method and the issue is that the country selector would still be bound by the height of the footer. A screen with this resolution is very unlikely, but in the image below I set the height so that more content other than the footer is visible and setting the overlay's height to 100vh doesn't actually take 100% of the viewport but rather 100% of the footer's height.

<img width="208" alt="Screenshot 2023-12-11 at 12 53 01 PM" src="https://github.com/Shopify/dawn/assets/54807699/e6000556-c135-4741-b08a-17461727cdcc">

Reply:
Uh, interesting 🤔
In my example the footer didn't take the whole screen (you can see the Subscribe panel but also the end of the section above, with the Shop all button.

In order to preserve the animation for desktop, would you then instead consider changing the CSS?
Currently [Dawn has this on line 3289](https://github.com/hamideha/dawn/blob/hamideha/selector/assets/base.css#L3289)
```css
.scroll-trigger.animate--slide-in {
    transform: translateY(2rem);
  }
```

We could add something to disable that transform in the footer and on mobile? Using the !important doesn't feel great but is necessary because the animation css kicks in and overrides it otherwise 😞
```css
.scroll-trigger.animate--slide-in {
    transform: translateY(2rem);
  }

@media screen and (max-width: 749px) {
  .footer__content-bottom.scroll-trigger.animate--slide-in {
    transform: none !important;
  }
}
```

Reply:
Ah yes that works and is a cleaner solution. Thank you!

Reply:
So am I understanding correctly the comments and the animation class should be re added ?

Reply:
I haven't added them back because testing on other WebKit browsers, specifically on iOS versions lower than 17 
```css
.footer__content-bottom.scroll-trigger.animate--slide-in {
    transform: none !important;
}
```
doesn't work. I'm still trying to figure out so I haven't added them back yet.